pyproject.toml

```diff
@@ -18,30 +18,36 @@ dependencies = [
     "sqlfluff == 2.0.2",
     "toml >= 0.10"
 ]
-description = "SQL generation for cumulus clinical studies"
+description = "Clinical study SQL generation for data derived from bulk FHIR"
 readme = "README.md"
 license = { text="Apache License 2.0" }
+home-page = "https://smarthealthit.org/cumulus-a-universal-sidecar-for-a-smart-learning-healthcare-system/"
+keywords = ["FHIR", "SQL", "Health Informatics"]
 classifiers = [
     "License :: OSI Approved :: Apache Software License",
     "Operating System :: OS Independent",
     "Programming Language :: Python :: 3",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
-
-[project.scripts]
-cumulus-library = "cumulus_library.cli:main"
-
-[build-system]
-requires = ["flit_core >=3.4,<4"]
-build-backend = "flit_core.buildapi"
-
 [project.optional-dependencies]
-
 dev = [
     "black",
     "pylint",
 ]
-
 test = [
     "pytest",
 ]
+
+[project.urls]
+Documentation = "https://docs.smarthealthit.org/cumulus/"
+Source = "https://github.com/smart-on-fhir/cumulus-library-core"
+
+[project.scripts]
+cumulus-library = "cumulus_library.cli:main"
```

Comment:
We talked about this a little on slack, but just putting it here for posterity - I think this needs more discussion. It's really going against convention to have a pip package install the binary `cumulus-library` and the module `cumulus_library` but not have the package name be `cumulus-library`.

That's bad for discoverability, and it's bad namespace-management conventions. I know we talked about having other studies be like `cumulus-library-<study>` and that's fine. But the core package should be, IMO, `cumulus-library`.

Let's chat more about this.

Reply:
ok, the project name is now reverted to `cumulus-library`.